tests/MongoDB.Driver.Tests/Specifications/client-side-encryption/prose-tests/ClientEncryptionProseTests.cs

```diff
@@ -1522,38 +1525,45 @@ void KmsProviderEndpointConfigurator(string kmsProviderName, Dictionary<string,
             }
         }
 
+        [Trait("Category", "CsfleGCPKMS")]
         [SkippableTheory]
         [ParameterAttributeData]
         public void OnDemandCredentials(
-            [Values("aws")] string kmsProvider,
-            [Values(false, true)] bool envVariablesSet,
+            [Values("aws", "gcp")] string kmsProvider,
+            [Values(false, true)] bool expectedEnvironment,
```

Comment:
`envVariablesSet` is clearer to me than `expectedEnvironment` though it should probably be named `environmentVariablesSet`.

Reply:
Fair point about it being about more than just environment variables. `expectedEnvironment` works for me.